addons/mail/static/src/models/thread/thread.js

```diff
@@ -1995,6 +1996,12 @@ registerModel({
         guestMembers: many('Guest', {
             sort: '_sortGuestMembers',
         }),
+        /**
+         * States whether the current user has read access for this thread.
+         */
+        hasAccess: attr({
+            default: true,
```

Comment:
also default from here could be removed if following my suggestion to make the python return true when appropriate